sdk-core/message.md

```diff
@@ -3,14 +3,38 @@
 ```
 dto Message:
     data: bytes;
-    signature: bytes;
+    signature: Optional[bytes];
+    address: Optional[IAddress];
+    version: Optional[int]
```

Comment:
I think in the specs we should not mention that they are optional. In (some) implementations they will be, yes - partly for historical reasons (legacy / avoiding some breaking changes), partly for practical reasons.